storm-core/test/resources/log4j2-test.xml

```diff
@@ -25,7 +25,8 @@
     <Loggers>
         <!-- suppress ERROR org.apache.storm.blobstore.BlobStoreUtils - Could not update the blob with key: key when testing -->
         <Logger name="org.apache.storm.blobstore" level="FATAL" />
-        <Logger name="org.apache.zookeeper" level="WARN"/>
+        <Logger name="org.apache.storm.shade.org.apache.zookeeper" level="WARN"/>
```

Comment:
Could you verify that this change makes sense on 1.x? I think it is here because master has the shaded-deps module, but in 1.x the shading happens as part of building storm-core, so I'm not sure the test classpath is affected by the relocation.

Reply:
@srdo I think we need to use origin package name. I missed here. Fixed.